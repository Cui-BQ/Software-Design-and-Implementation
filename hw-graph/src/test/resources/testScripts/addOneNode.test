# Add one node and ListNode

# Create a graph
CreateGraph graph1

# Add node n1
AddNode graph1 n1

# graph1 lists nodes
ListNodes graph1