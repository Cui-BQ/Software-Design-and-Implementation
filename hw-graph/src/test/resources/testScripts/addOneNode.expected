# Add one node and ListNode

# Create a graph
created graph graph1

# Add node n1
added node n1 to graph1

# graph1 lists nodes
graph1 contains: n1